libcudacxx/test/support/test_execution_policies.h

```diff
@@ -0,0 +1,55 @@
+//===----------------------------------------------------------------------===//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+// SPDX-FileCopyrightText: Copyright (c) 2025 NVIDIA CORPORATION & AFFILIATES
+//
+//===----------------------------------------------------------------------===//
+
+#ifndef TEST_SUPPORT_TEST_EXECUTION_POLICIES
+#define TEST_SUPPORT_TEST_EXECUTION_POLICIES
+
+#include <cuda/std/cstdlib>
+#include <cuda/std/execution>
+#include <cuda/std/type_traits>
+#include <cuda/std/utility>
+
+#include "test_macros.h"
+
+#define EXECUTION_POLICY_SFINAE_TEST(function)                                                                          \
+  template <class, class...>                                                                                            \
+  struct sfinae_test_##function##_impl : cuda::std::true_type                                                           \
+  {};                                                                                                                   \
+                                                                                                                        \
+  template <class... Args>                                                                                              \
+  struct sfinae_test_##function##_impl<cuda::std::void_t<decltype(cuda::std::function(cuda::std::declval<Args>()...))>, \
```

Comment:
suggestion: I've spent maybe 10 minutes tryign to parse `cuda::std::function` CTAD usage here to realize that it's not `cuda::std::function`.. Consider capitalizing `function` to make it a bit easier to parse:
```suggestion
  struct sfinae_test_##function##_impl<cuda::std::void_t<decltype(cuda::std::FUNCTION(cuda::std::declval<Args>()...))>, \
```